Resolutions/ChildOfLight_Resolution/rules.txt

```diff
@@ -108,26 +107,28 @@ formats = 0x01a
 overwriteWidth = ($width/$gameWidth) * 512
 overwriteHeight = ($height/$gameHeight) * 512
 
-[TextureRedefine] # Half-res
+[TextureRedefine] # Half-Res
+width = 640
+height = 368
+tileModesExcluded = 0x001
+overwriteWidth = ($width/$gameWidth) * 640
+overwriteHeight = ($height/$gameHeight) * 368
+
+[TextureRedefine]
 width = 640
 height = 360
-formats = 0x001
 tileModesExcluded = 0x001
 overwriteWidth = ($width/$gameWidth) * 640
 overwriteHeight = ($height/$gameHeight) * 360
 
-# Screen gradient
-
-[TextureRedefine]
-width = 426
+[TextureRedefine] # Screen gradient
+width = 448
 height = 240
-formats = 0x01a
-overwriteWidth = ($width/$gameWidth) * 426
+overwriteWidth = ($width/$gameWidth) * 448
 overwriteHeight = ($height/$gameHeight) * 240
 
 [TextureRedefine]
-width = 448
+width = 426
 height = 240
-formats = 0x01a
-overwriteWidth = ($width/$gameWidth) * 448
+overwriteWidth = ($width/$gameWidth) * 426
 overwriteHeight = ($height/$gameHeight) * 240
```

Comment:
Personally think these are mostly just preference things, do I do agree that putting the `# Screen gradient` on the right might be a little cleaner. 🤷

Reply:
Yes, I do prefer having them ordered from biggest to smallest textures because it makes it easier to search for especially when there are a lot of textures being up-scaled. If you don't like all the other changes I made to this pack, I can easily just revert them. I just wanted to clean this pack specifically since I based one of my packs off of this one since Rayman Legends runs on the same engine.

Reply:
I think reverting the `formats = 0x1a` here would resolve this issue.

Reply:
I think it's just a personal preference thing. I like to make my graphic packs very orderly and very defined, but I'm not saying that it's the right way to make graphic packs.